packages/tic-tac-toe/config/dotenv.js

```diff
@@ -6,6 +6,19 @@ const path = require('path');
 const fs = require('fs');
 
 function configureEnvVariables(env) {
+  // State Channel Environment
+  // Intended usage is a single file in monorepo root defining configuration for multiple packages
+  const SC_ENV = process.env.SC_ENV;
```

Comment:
Any reason not to import this from @statechannels/devtools?

Reply:
`devtools` requires the env vars directly in this function.
In Ember I use an Ember Addon to inject the env vars during runtime.
If the devtools function was split up into two functions (one that returns a path and one that requires the env vars) I could utilize it.